tests/Java.Interop.Tools.JavaSource-Tests/SourceJavadocToXmldocGrammar.HtmlBnfTermsTests.cs

```diff
@@ -69,7 +69,11 @@ public void HyperLinkDeclaration ()
 
 			r = p.Parse ("<a href=\"AutofillService.html#FieldClassification\">field classification</a>");
 			Assert.IsFalse (r.HasErrors (), DumpMessages (r, p));
-			Assert.AreEqual ("\"AutofillService.html#FieldClassification\"&gt;field classification",
+			Assert.AreEqual ("field classification", r.Root.AstNode.ToString ());
+
+			r = p.Parse ("<a href='https://material.io/guidelines/components/progress-activity.html#progress-activity-types-of-indicators'>\nProgress & activity</a>");
```

Comment:
I see that this matches one of the error cases mentioned in the PR message:

> ```
> ## Unable to parse HTML element: <see href="https://material.io/guidelines/components/progress-activity.html#progress-activity-types-of-indicators">
> Progress & activity</see>
> System.Xml.XmlException: An error occurred while parsing EntityName. Line 2, position 11.
> ```

This is good!

I think we should *also* add tests for the *other* instances mentioned in the PR message, including `<a href="#BROKEN"> <code>broken</code></a>` and whatever caused the other failures. :-)

Reply:
Thanks good point, I've added tests that cover the rest of the failing cases which helped me realize that the changes to `InlineTagsBnfTerms.IgnorableDeclaration` from 1125 were missing here.